skcosmo/sample_selection/_base.py

```diff
@@ -346,3 +351,156 @@ def __init__(
             full=full,
             random_state=random_state,
         )
+
+
+class DirectionalConvexHull:
+    """
+    Transformer that performs Sample Selection by constructing a
+    Directional Convex Hull and determining the distance to the hull, as
+    outlined in:
+
+    A. Anelli, E. A. Engel, C. J. Pickard and M. Ceriotti,
+    Physical Review Materials, 2018.
+
+
+    Attributes
+    ----------
+
+    dch_dims      : list of ints
+                    Indices of columns in data containing features to be
+                    used for the directional convex hull construction
+    extra_dims    : list of ints
+                    Indices of columns in data containing high-dimensional
+                    features (i.e. those not used for the convex hull
+                    construction)
+    selected_idx_ : numpy.ndarray
+                    Indices of datapoints that form the vertices of the
+                    convex hull
+    interpolator  : functools.partial
+                    Partial function for interpolating features in the high-
+                    dimensional space (used in get_score)
```

Comment:
why is this a public attribute? Is this something the users should be able to use?

Reply:
Ah there is no reason. I will make it a private attribute.